lib/improver/tests/spotdata/spotdata/test_WriteOutput.py

```diff
@@ -39,9 +39,12 @@
 from iris.tests import IrisTest
 from iris.cube import Cube
 from iris.coords import DimCoord
+from iris import FUTURE
 
 from improver.spotdata.write_output import WriteOutput as Plugin
 
+FUTURE.netcdf_promote = True
```

Comment:
Should we have this in a with statement or is it ok to apply it to the whole module? Will this behaviour persist if this module gets imported?

Reply:
This is how it has been done in other unit tests but I will change it to a with statement.